### PR TITLE
DDCE-2631 - Update MODs to the latest versions of sbt

### PR DESCRIPTION
**Pull request by ryandac**
# DDCE-2631 - Update MODs to the latest versions of sbt

If the unit tests do not pass locally, delete the mongo database and re-run them.

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -9,12 +9,12 @@ object AppDependencies {
     "uk.gov.hmrc"                %% "bootstrap-frontend-play-28" % "5.20.0",
     "uk.gov.hmrc"                %% "auth-client"                % "5.8.0-play-28",
     "uk.gov.hmrc"                %% "play-frontend-hmrc"         % "0.94.0-play-28",
```
Comment (jassalrichy) [inline project/AppDependencies.scala:11]: Update  "uk.gov.hmrc: play-frontend-hmrc" to "3.13.0-play-28"

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -9,12 +9,12 @@ object AppDependencies {
...
-    "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-28"         % "0.60.0",
+    "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-28"         % "0.62.0",
     "com.github.pureconfig"      %% "pureconfig"                 % "0.17.1",
     "com.beachape"               %% "enumeratum-play"            % "1.7.0",
     "org.webjars.npm"            %  "accessible-autocomplete"    % "2.0.3",
-    "uk.gov.hmrc"                %% "play-language"              % "5.1.0-play-28",
```
Comment (jassalrichy) [inline project/AppDependencies.scala:16]: Should this not remain on version "5.1.0-play-28" ?
See:
https://catalogue.tax.service.gov.uk/dependencies/merchandise-in-baggage-frontend/0.539.0

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -9,12 +9,12 @@ object AppDependencies {
...
-    "com.softwaremill.quicklens" %% "quicklens"                  % "1.8.3",
```
Comment (jassalrichy) [inline project/AppDependencies.scala:17]: Should this not remain on version "1.8.3" ?

**Review by jassalrichy — CHANGES_REQUESTED** [review]
Comments added to PR

File: project/AppDependencies.scala (hunk 2 above)
Comment (ryandac) [reply project/AppDependencies.scala]: Not sure, but have reverted the change

**Pull request merged**